### ISSUE TITLE
Ibm_db installation error

### ISSUE THREAD

**Issue by suresht-113** [ibmdb/node-ibm_db#381]
i am newbie to node.js and want to connect db2 to node.js. i have downloaded the latest binaries from the node.js website and installed. when trying to perform npm install ibm_db, i get error as in attached document.  using the instruction to install for windows from this link https://github.com/ibmdb/node-ibm_db/blob/master/INSTALL.md
my system is window 7 and node.js version is 9.11.1
what am i missing?
Do i have to install Python as mentioned in the prerequisites in this link?  
https://www.npmjs.com/package/ibm_db
i am little confused. any help would be welcome
![image](https://user-images.githubusercontent.com/38451618/38851278-bfc69e54-4232-11e8-927f-cbffeae97084.png)

**Comment by No9** [comment]
Hi @suresht-113 
That looks like your machine cannot find the server public.dhe.ibm.com to download 
https://public.dhe.ibm.com/ibmdl/export/pub/software/data/db2/drivers/odbc_cli/ntx64_odbc_cli.zip 

The server seems to be up at the moment and looking at the prior npm registry error aound ssl it may be that you have a proxy server that isn't functioning quite as it should??

**Comment by No9** [comment]
Hi @suresht-113 
Here are instructions for installing with the driver already downloaded
https://github.com/ibmdb/node-ibm_db#important-environment-variables-and-download-essentials 

I also note that you are running the commands in a console running as Administrator 
Are you sure that user has access to the internet?

**Comment by bimalkjha** [comment]
@suresht-113 Are you still facing this issue? Could you retry and confirm?  Thanks.

**Comment by bimalkjha** [comment]
@suresht-113 Yes, your license file is not correct. You need a db2connect license file something like db2con*.lic  and copy it under ibm_db/installer/clidriver/license folder. Please contact your sysadmin to get db2connect license or contact IBM Support to get it. Thanks.

**Comment by suresht-113** [comment]
thanks a lot. I will check with our admin
